src/wizard/capacitor/index.md

```diff
@@ -76,12 +68,16 @@ Sentry.init({
 });
 ```
 
-### Verifying Your Setup
+### Send Your First Event
 
-Great! Now that you’ve completed setting up the SDK, maybe you want to quickly test out how Sentry works. Start by capturing an exception:
+This snippet includes an intentional error, so you can test that everything is working as soon as you set it up:
```

Comment:
There's no intentional error though, it's just capturing an exception (with a string).

What about a `captureMessage` then?

Reply:
Would you like me to change the exception to a captureMessage? Or are you thinking it would be smarter to switch the verbiage back to the earlier version?

Reply:
We can merge it as-is for now, just need to get a package on npmjs first. THanks!